packages/create-nx-workspace/bin/package-manager.ts

```diff
@@ -7,7 +7,9 @@ import { join } from 'path';
  * we duplicate the helper functions from @nrwl/workspace in this file.
  */
 
-export type PackageManager = 'yarn' | 'pnpm' | 'npm';
+const packageManagerList = ['npm', 'pnpm', 'yarn'] as const;
```

Comment:
The order of the list has changed. Can you please ensure the order is:
1. `pnpm`
2. `yarn`
3. `npm`